workspaces/bazaar/plugins/bazaar-backend/README.md

```diff
@@ -1,52 +1,18 @@
-# Bazaar Backend
+# bazaar-backend
```

Comment:
Not sure why we changed the header and removed the Welcome sentence, that seemed fine to have.